go.mod

```
@@ -99,11 +99,12 @@ require (
 	gonum.org/v1/gonum v0.14.0
 	google.golang.org/grpc v1.59.0
 	google.golang.org/protobuf v1.31.0
-	gopkg.in/guregu/null.v2 v2.1.2
 	gopkg.in/guregu/null.v4 v4.0.0
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1
 )
 
+require gopkg.in/guregu/null.v2 v2.1.2
```

We should probably drop v2 but just formatting for now.
```suggestion
	gopkg.in/guregu/null.v2 v2.1.2
	gopkg.in/guregu/null.v4 v4.0.0
	gopkg.in/natefinch/lumberjack.v2 v2.2.1
)
```